<launch>
    <node name="gicp_align" pkg="fast_gicp" type="gicp_align" args="/home/xyw/catkin_ws/src/fast_gicp/data/251370668.pcd /home/xyw/catkin_ws/src/fast_gicp/data/251371071.pcd" output="screen"> 
        <param name="litamin_source_size" value="5" />
        <param name="litamin_target_size"  value="5" />
        <param name="pcd_1"  value= "/home/xyw/catkin_ws/src/fast_gicp/data/003121.pcd"/>
        <param name="pcd_2"  value= "/home/xyw/catkin_ws/src/fast_gicp/data/003122.pcd"/>
        <param name="vgicp_direct_method" value="DIRECT1"/>
        <param name="vgicp_resolution" value="1.0"/>
        <param name= "transform_epsilon" value="1e-2"/>
        <param name="MaxCorrespondenceDistance" value="0.5"/>
        <param name ="num_threads" value="4"/>
    </node>
    
</launch>